examples/lang/virt2.mcl

```diff
@@ -29,26 +29,24 @@ virt "mgmt4" {
 	state => "running",
 	transient => false,
 	boot => ["hd", ],
-	# can't add this part until we fix the unification bug
-	#disk => [
-	#	struct{
-	#		source => "~/.local/share/libvirt/images/fedora-23-scratch.qcow2",
-	#		type => "qcow2",
-	#	},
-	#],
-	# add the rest for unification bug
-	#osinit => "",
-	#cdrom => [
-	#],
-	#network => [
-	#],
-	#filesystem => [
-	#],
-	#auth => struct{
-	#	username => "",
-	#	password => "",
-	#},
-	#hotcpus => true,	# this is the default
-	#restartondiverge => "",
-	#restartonrefresh => false,
+	disk => [
+		struct{
+			source => "~/.local/share/libvirt/images/fedora-23-scratch.qcow2",
+			type => "qcow2",
+		},
+	],
+	osinit => "",
```

Comment:
Out of curiosity, did you test all this out? ;) Don't worry about it though, we can merge as is =D

Reply:
I tried running the example and the  unification works properly now, as expected, but the resources failed due to missing files. I didn't debug further than that